_el/exercises.md

```diff
@@ -171,52 +98,34 @@ search for it in the list of options.
 >            except:
```

Comment:
Εδώ υπάρχει μάλλον κάποιο πρόβλημα. Στο πρωτότυπο βλέπω και άλλες γραμμές:
A)     vals = [-1, 0, 1]
B)     inverse_sum = 0
       try:
           for v in vals:
C)             inverse_sum += 1/v
       except:
D)         pass

Reply:
Ευχαριστώ για το σχόλιο, αλλά δεν βρίσκω κάτι διαφορετικό στις δύο εκδόσεις. Σε ποιά σειρά ακριβώς υπάρχει το πρόβλημα?

Reply:
Μετά το try: 
υπάρχουν οι γραμμές:
for v in vals:
C) inverse_sum += 1/v

Δικό μου λάθος, όταν έγραψα πρωτότυπο, δεν εννοούσα στο Github, αλλά στο βιβλίο... Στο Github, τα βλέπω και εωγ ίδια, αλλά πρέπει να υπάρχει και η επιλογη c), και βέβαια πρέπει και να υπάρχουν όλες οι σειρές για να βγαίνει ο κώδικας...